src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -11,9 +22,13 @@ import { pageLayout } from 'utils/form';
 import styles from '../../index.module.scss';
 
 const { Item } = Form;
-const { Option } = Select;
 const { Panel } = Collapse;
 
+const { Option } = AntdSelect;
+const Select = withWritableInput(AntdSelect);
```

Comment:
you copied and pasted `withWritableInput` through out the application, it would be better if you just make components that wraps this, and replace all instances of the antd components with this component